### PR TITLE
[IMPROVE] Visibility of the formatting tools improvised 

### PR DESCRIPTION
**Pull request by PiyushDhamane**
## Proposed changes (including videos or screenshots)

image
Please check the textarea, hover effect and icon size. 

## Issue(s)

#21108 fixed

### CODE REVIEW

**Comment by PiyushDhamane** [comment]
@Sing-Li @aditya-mitra @yash-rajpal please have a look at this

File: package-lock.json [hunk 1]
```diff
@@ -37432,4 +37432,4 @@
 			"dev": true
 		}
 	}
-}
+}
```
Comment (aditya-mitra) [inline package-lock.json:37435]: can you please undo this change?

Comment (PiyushDhamane) [reply package-lock.json:37435]: @aditya-mitra I'm not sure why is this change shown even though I didn't commit it. If you see it says, the brace deleted from 37435 and added again to the same line. 
This is a problem I faced in my other commit too.

File: app/ui-message/client/messageBox/messageBox.html [hunk 2]
```diff
@@ -23,7 +23,7 @@
 					<span class="rc-message-box__icon emoji-picker-icon {{#unless isEmojiEnabled}}emoji-picker-icon--disabled{{/unless}} js-emoji-picker" aria-haspopup="true">
 						{{> icon block="rc-input__icon-svg" icon="emoji"}}
 					</span>
-					<textarea aria-label="{{_ 'Message'}}" name="msg" maxlength="{{maxMessageLength}}" placeholder="{{_ 'Message'}}" rows="1" class="rc-message-box__textarea js-input-message"></textarea>
+					<textarea aria-label="{{_ 'Message'}}" name="msg" maxlength="{{maxMessageLength}}" placeholder="{{_ 'Message(check the icons below to beautify your text)'}}" rows="1" class="rc-message-box__textarea js-input-message"></textarea>
```
Comment (ggazzo) [inline app/ui-message/client/messageBox/messageBox.html:26]: sorry this not a valid I18n Key

**Review by ggazzo — CHANGES_REQUESTED** [review]

**Comment by dudanogueira** [comment]
A lot of changes has changed since the React migration. 
Please, feel free to repoen or  craft a new PR with the updated codebase.
Closing for now.
Thanks for your contribution!
